network/manager.go

```diff
@@ -113,6 +125,10 @@ func NewNetworkManager(nl netlink.NetlinkInterface, plc platform.ExecClient, net
 func (nm *networkManager) Initialize(config *common.PluginConfig, isRehydrationRequired bool) error {
 	nm.Version = config.Version
 	nm.store = config.Store
+	if config.Stateless == true {
+		nm.SetStatelessCNIMode()
```

Comment:
We are bypassing the error returned by `SetStatelessCNIMode`, we should return that error

Reply:
addressed in the next commit.